go-controller/pkg/factory/factory.go

```diff
@@ -1140,7 +1140,7 @@ func noAlternateProxySelector() func(options *metav1.ListOptions) {
 func WithUpdateHandlingForObjReplace(funcs cache.ResourceEventHandler) cache.ResourceEventHandlerFuncs {
 	return cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
-			funcs.OnAdd(obj)
+			funcs.OnAdd(obj, true)
```

Comment:
nit: I guess I would set this to false, inline with the other places where we are setting it to false as well.

This value is not going anywhere anyway.

I think this value isn't going anywhere since we are not

Reply:
@jcaamano , [here](https://github.com/openshift/ovn-kubernetes/pull/1756) is the PR to address this.... finally :)